features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_en.json

```diff
@@ -147,8 +147,10 @@
     "Description": "Description",
     "Design": "Design",
     "Design API": "Design API",
-    "Design New API": "Design New API",
-    "Design and prototype a new API.": "Design and prototype a new API.",
+    "Design New REST API": "Design New REST API",
+    "Design and prototype a new REST API.": "Design and prototype a new REST API.",
+    "Design New Websocket API": "Design New Websocket API",
```

Comment:
Put these constance in the local_default.json file as well. Otherwise for any other language other than 'en' will have null values in the UI